tests/cli/test_cli.py

```diff
@@ -30,6 +31,8 @@ def test_data_convert_help(run: Callable[..., RunResult]):
         assert output.outlines[i] == line
 
 
+@pytest.mark.linux
+@pytest.mark.darwin
 def test_version_print_lines(run: Callable[..., RunResult]):
```

Comment:
why does it not work on Windows? If the output is formatted differently, should we add a Windows only test then?

Reply:
+1 is there any way this can be checked on windows?